cli/app.go

```diff
@@ -425,32 +425,114 @@ var app = &cli.App{
 					Subcommands: []*cli.Command{
 						{
 							Name:  "add",
-							Usage: "adds binary data with file IDs in a single org and location to dataset",
-							UsageText: fmt.Sprintf("viam data dataset add <%s> <%s> <%s> <%s> [other options]",
-								datasetFlagDatasetID, dataFlagOrgID, dataFlagLocationID, dataFlagFileIDs),
-							Flags: []cli.Flag{
-								&cli.StringFlag{
-									Name:     datasetFlagDatasetID,
-									Usage:    "dataset ID to which data will be added",
-									Required: true,
-								},
-								&cli.StringFlag{
-									Name:     dataFlagOrgID,
-									Usage:    "org ID to which data belongs",
-									Required: true,
+							Usage: "adds binary data either by IDs or filter to dataset",
+							Subcommands: []*cli.Command{
+								{
+									Name:  "ids",
+									Usage: "adds binary data with file IDs in a single org and location to dataset",
+									UsageText: fmt.Sprintf("viam data dataset add-by-id <%s> <%s> <%s> <%s> [other options]",
```

Comment:
While you're here, we're about to update this across Data and Fleet: https://viam.atlassian.net/browse/DATA-2189

Could you update these to --%s=<%s> throughout, for at least what's been newly added in this PR?

Reply:
Update here and throughout all the dataset stuff

Reply:
Thank you! Could you test that that prints locally as expected?

Reply:
Yes, prints as expected :)